src/app/app-routing.module.ts

```diff
@@ -18,6 +19,7 @@ const routes: Routes = [
   { path: 'ticket-generator', component: TicketGeneratorComponent },
   { path: 'qc-timer', component: QcTimerComponent },
   { path: 'help', component: HelpComponent },
+  { path: 'weatherView', component: WeatherViewComponent },
```

Comment:
do we want to do `weather-view` here instead?

Reply:
ok